common/spec/dependabot/config/ignore_condition_spec.rb

```diff
@@ -287,6 +288,24 @@ def expect_ignored(versions)
         end
       end
 
+      context "with a semver dependency, but according to another package manager" do
+        let(:dependency_version) { "v11.0.14" }
+
+        context "with ignore_major_versions" do
+          let(:update_types) { ["version-update:semver-major"] }
+
+          it "ignores expected versions" do
+            expect_allowed(["11"])
+            expect_ignored(["17"])
+            expect_allowed([dependency_version])
+          end
+
+          it "returns the expected range" do
+            expect(ignored_versions).to eq([">= 12.a"])
```

Comment:
What does the `.a` in this version string represent? Just a placeholder?

Reply:
It's a RubyGems thing. ">= 12.a" is essentially the same thing as ">= 12", except that it forces inclusion of prereleases in the range, while ">= 12" would not include prereleases of version 12, like "12.0.0.beta".